examples/torch/mtsac_MT10.py

```diff
@@ -0,0 +1,100 @@
+"""MTSAC implementation based on Metaworld. Benchmarked on MT10.
+
+https://arxiv.org/pdf/1910.10897.pdf
+"""
+import click
+from metaworld.benchmarks import MT10
+import numpy as np
+from torch import nn
+from torch.nn import functional as F
+
+from garage import wrap_experiment
+from garage.envs import GarageEnv, MultiEnvWrapper, normalize
+from garage.experiment import deterministic, LocalRunner
+from garage.replay_buffer import SimpleReplayBuffer
+from garage.sampler import LocalSampler
+from garage.torch.algos import MTSAC
+from garage.torch.policies import TanhGaussianMLPPolicy
+from garage.torch.q_functions import ContinuousMLPQFunction
+import garage.torch.utils as tu
+
+
+@click.command()
+@click.option('--seed', 'seed', type=int, default=1)
+@click.option('--use_gpu', 'use_gpu', type=bool, default=False)
+@click.option('--gpu', '_gpu', type=int, default=0)
+@wrap_experiment(snapshot_mode='none')
+def mt10_mtsac_normalize_reward(ctxt=None, seed=1, use_gpu=False, _gpu=0):
+    """Train MTSAC with MT10 environment.
+
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+        use_gpu (bool): Used to enable ussage of GPU in training.
+        _gpu (int): The ID of the gpu (used on multi-gpu machines).
+
+    """
+    deterministic.set_seed(seed)
+    runner = LocalRunner(ctxt)
+    task_names = MT10.get_train_tasks().all_task_names
+    mt10_train_envs = MultiEnvWrapper([
```

Comment:
can you manually format this so it looks less strange? or just construct your list of envs in a different block before this one.